### DISCUSSION TITLE
Remove permissions on android

### DISCUSSION THREAD

**Discussion by phamcham** [yasirkula/UnityNativeGallery#272] [Q&A] [answered]
Hi, I don't want to have android.permission.WRITE_EXTERNAL_STORAGE on android, so is there any way to this plugin work only on ios?

**Comment by yasirkula** [comment, marked as answer]
Hi! You can delete NativeGallery.aar from Plugins/NativeGallery/Android folder and never call any NativeGallery functions on Android so that the plugin works on iOS only while not introducing any new permissions on Android.
